FinanceHelper/FinanceHelper/Program.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using FinanceHelper.BLL;
+using FinanceHelper.Common;
+using FinanceHelper.Common.Entity;
+using FinanceHelper.DAL;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace FinanceHelper
+{
+    class Program
+    {
+        public static readonly IServiceProvider container = new ContainerBuilder().Build();
+
+        static void Main(string[] args)
+        {
+            IDistributor distributor = container.GetService<IDistributor>();
```

Comment:
You need to make distributor field in Program class in this case and initialize this in constructor.